### PR TITLE
feat: add medals to exercise done

### PR DESCRIPTION
**Pull request by Gamer92000**

### CODE REVIEW

File: src/locales/de/translation.json [hunk 1]
```diff
@@ -182,6 +182,10 @@
     "accuracy": "Genauigkeit",
     "speed": "Schnelligkeit",
     "score-short": "Pkt",
-    "current-set": "Aktuelles Set"
+    "current-set": "Aktuelles Set",
+    "medal": "Hier könnte deine Medaille stehen!",
```
Comment (Laennart) [inline src/locales/de/translation.json:186]: A little suggestion for this message: (To make it more gamified)
```suggestion
    "medal": "Leider hast du dieses Mal keine Medaille verdient!",
```

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -182,6 +182,10 @@
     "accuracy": "accuracy",
     "speed": "speed",
     "score-short": "Pts",
-    "current-set": "Current Set"
+    "current-set": "Current Set",
+    "medal": "Here could be your medal!",
```
Comment (Laennart) [inline src/locales/en/translation.json:186]:
```suggestion
    "medal": "Sadly you did not earn a medal this time!",
```

**Review by Laennart — CHANGES_REQUESTED** [review]
Works nicely! Just two minor suggestions for improvement! :)

**Review by Laennart — APPROVED** [review]
🏅

**Review by Laennart — APPROVED** [review]

**Review by Y4nnikH — APPROVED** [review]
Looking good!

**Pull request merged**